### DISCUSSION TITLE
how to Minify html such as WebMarkupMin library?

### DISCUSSION THREAD

**Discussion by Mostafa-Moafi** [dnnsoftware/Dnn.Platform#5727] [General]
Hi
How can I Minify html such as WebMarkupMin library?

more information:

[WebMarkupMin](https://github.com/Taritsyn/WebMarkupMin)

**Comment by bdukes** [comment]
I'm not familiar with this library, and I'm not sure that minification of HTML is going to provide a significant benefit when the content is already gzipped. However, it looks like you could probably add the assemblies from [the WebMarkupMin.AspNet4.HttpModules NuGet package](https://www.nuget.org/packages/WebMarkupMin.AspNet4.HttpModules/) and then register one of its HTTP modules in the `web.config`, and see if it affects the output of the pages.

**Reply by Mostafa-Moafi** [reply]
I install this package but errors that it needs to .netstandard v2.1 and I can't test it.
Thanks for your attention

**Reply by bdukes** [reply]
You'll need to make sure you're installing it into a project that targets .NET 4.7.2, so that you get the correct version of the dependencies.

**Reply by Mostafa-Moafi** [reply]
Yes of course, but the [WebMarkupMin](https://github.com/Taritsyn/WebMarkupMin) Library is dependent on .NET 4.5 and does not support .NET 4.7.2

**Reply by bdukes** [reply]
It should support .NET 4.5 _and higher_

**Comment by mitchelsellers** [comment]
DNN used to have a minify HTML process, but it created more problems than it helped, and as @bdukes mentioned I believe the usage of gzip etc which is managed by IIS etc it isn't worth it
